components/esptool_py/Kconfig.projbuild

```diff
@@ -14,10 +14,18 @@ choice ESPTOOLPY_BAUD
 		Default baud rate to use while communicating with the ESP chip. Can be overridden by
 		setting the ESPBAUD variable.
 
+config ESPTOOLPY_BAUD_57600B
+	bool "57600 baud"
 config ESPTOOLPY_BAUD_115200B
 	bool "115200 baud"
 config ESPTOOLPY_BAUD_230400B
 	bool "230400 baud"
+config ESPTOOLPY_BAUD_256000B
+	bool "256000 baud"
+config ESPTOOLPY_BAUD_460800B
+	bool "460800 baud"
+config ESPTOOLPY_BAUD_512000B
+	bool "512000 baud"
```

Comment:
Why need these baud rates? Does the higher baud rate not work for you?

Reply:
No, I have to use 460800 on my NodeMCU dev board. It has been useful to diagnose boards with defects earlier too.

Reply:
Can you use `( ) Other baud rate` in menuconfig to set the baud rate? Listing all baud rates maybe let the menu too complex. Thanks